### ISSUE TITLE
Windows Error: Program 'git' must be installed

### ISSUE THREAD

**Issue by cope** [simonrenoult/code-complexity#46]
In WSL it works like a charm, but in Windows I get the `Error: Program 'git' must be installed` error, because `'which' is not recognized as an internal or external command` 😞 

I tried using `Set-Alias which where.exe` and it works on the command line:
```
PS C:\windows\system32> which git
C:\Program Files\Git\cmd\git.exe
```

But I still get 
```
'which' is not recognized as an internal or external command,
operable program or batch file.
Error: Program 'git' must be installed
```
😞

**Comment by cope** [comment]
I think that `assertGitIsInstalled()` is not needed. If git is not install, trying to call it would throw an error.

**Comment by kireerik** [comment]
Yes, I agree. We should remove it.  @simonrenoult What do you think?
